test/common/README.md

```diff
@@ -431,6 +432,48 @@ Decrements the `Countdown` counter.
 Specifies the remaining number of times `Countdown.prototype.dec()` must be
 called before the callback is invoked.
 
+## CPU Profiler module
+
+The `cpu-prof` module provides utilities reelated to CPU profiling tests.
+
+### env
+
+* Default: { ...process.env, FIB, NODE_DEBUG_NATIVE: 'INSPECTOR_PROFILER' }
+
+Environment variables used in profiled processes. FIB will be set to `40` on
+Windows and `30` elsewhere.
+
+### getCpuProfiles(dir)
+
+* `dir` {string} The directory containing the CPU profile files.
+* return [&lt;string>]
+
+Returns an array of all `.cpuprofile` files found in `dir`.
+
+### getFrames(file, suffix)
+
+* `file` {string} Path to a `.cpuprofile` file.
+* `suffix` {string} Suffix of the URL of call frames to retrieve.
+* returns { frames: [&lt;Object>], nodes: [&lt;Object>] }
+
+Returns an object containing an array of the relevant call frames and an array
+of all the profile nodes.
+
+### kCpuProfInterval
+
+* Default: 50
```

Comment:
Likewise I think we should be fine by just saying something like

```
A reasonable CPU profiler sampling interval for `test/fixtures/workload/fibonacci.js`
```

Reply:
Since those are both specific to the test fixture, would it make sense to pull them out of the module and set their values in the fixture itself?

Reply:
Er, OK, maybe not this one, but the FIB one for sure.

Reply:
Are you saying we shouldn't mention the default value? Or that we should use your suggested text instead of `Sampling interval in microseconds.`? Or both?

Reply:
I meant we should not mention the specific default value as we'll need to update the docs whenever the value is updated...even though the value is just empirical.

Reply:
OK, I removed mention of the default value.